test/abstractarray.jl

```diff
@@ -840,3 +840,9 @@ end
 @testset "checkbounds_indices method ambiguities #20989" begin
     @test Base.checkbounds_indices(Bool, (1:1,), ([CartesianIndex(1)],))
 end
+
+@testset "ImageCore #40" begin
+    Base.convert(::Type{Array{T,n}}, a::Array) where {T<:Number,n} =
```

Comment:
is the convert method wrong then or ? if this changes behavior, should this be careful to always stay the last thing that runs in an isolated test, or try to replicate with a test local type?